pkg/manager/member/tikv_member_manager.go

```diff
@@ -315,6 +315,36 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster) (*apps.StatefulSet, e
 		})
 	}
 
+	sysctls := "sysctl -w"
+	var initContainers []corev1.Container
+	if tc.Spec.TiKV.Annotations != nil {
+		init, ok := tc.Spec.TiKV.Annotations[label.AnnSysctlInit]
+		if ok && (init == label.AnnSysctlInitVal) {
+			if tc.Spec.TiKV.PodSecurityContext != nil && len(tc.Spec.TiKV.PodSecurityContext.Sysctls) > 0 {
+				for _, sysctl := range tc.Spec.TiKV.PodSecurityContext.Sysctls {
+					sysctls = sysctls + fmt.Sprintf(" %s=%s", sysctl.Name, sysctl.Value)
+				}
+				privileged := true
+				initContainers = append(initContainers, corev1.Container{
+					Name:  "init",
+					Image: controller.GetSlowLogTailerImage(tc),
```

Comment:
I suggest using an annotation to configure the init container image, and also set a default image when the annotation is not available.

Reply:
I prefer a dedicate field instead of annotation if we really want it to be configurable, for troubleshooting purpose, changing the image via `kubectl edit` is more straightforward.

Reply:
I think it's unnecessary to introduce user interface change for this image, it does not matter what it is and for TiDB version >= v2.1.8, slow log tailer must be enabled to check slow log.